docker.go

```diff
@@ -15,13 +15,45 @@ import (
 	"github.com/docker/docker/client"
 	"github.com/docker/docker/pkg/jsonmessage"
 	"github.com/docker/docker/pkg/term"
+	"github.com/docker/go-connections/nat"
 	"github.com/gorilla/websocket"
 	"github.com/sirupsen/logrus"
 )
 
+type containerInfo struct {
+	dockerImage string
+	port        string
+}
+
+func generatePort(portStr string) (nat.Port, error) {
+	if portStr == "" {
+		return "", nil
+	}
+	port, err := nat.NewPort("tcp", portStr)
+	if err != nil {
+		return "", err
+	}
+	portInt := port.Int()
+	if portInt < 36100 || portInt > 36110 {
+		return "", fmt.Errorf("port not in range [36100, 36110], given: %d", portInt)
```

Comment:
Maybe a comment saying this range comes from the port range let open by container-escape-bounty:
https://github.com/kinvolk/container-escape-bounty/pull/19

Reply:
done